xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/tables.less

```diff
@@ -62,7 +62,7 @@
 // headers on the left
 .responsive-table-tbody-tr-tdth() {
   border: none;
-  border-bottom: 1px solid @table-border-color;
+  border-bottom: 1px dotted @table-border-color;
```

Comment:
It doesn't impact only LD, does it?

Reply:
No, it impacts all `.responsive-table`s.

See https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/SpecialCSSClasses/#HClassesfortheresponsiveness

However I just got a look at an example of such a table in XS, the inline history:
![image](https://github.com/user-attachments/assets/8409e5b7-5380-4838-bff4-96a11290ac5e)
before changes in the PR on the left, after changes in the PR on the right.
IMO it's an improvement for this use case.

Making the separation between entries (one line) and different fields of the same lines more different is something that would improve the user experience for all of these responsive tables in my opinion.
What do you think?

Reply:
By the way, this class should not be mistaken for `.table-responsive` which comes from bootstrap :upside_down_face:

Reply:
Thanks for the check, that's better even if that UI is almost unusable IMO: I've no idea when looking to it what's the purpose of the first 2 radio buttons... But anyway, not related to your PR.